openjdk8/src/test/workspace/setup-java.bash

```diff
@@ -101,4 +101,19 @@ if [ "$TEST_MIN_HEAP" != "OK" -o "$TEST_MAX_HEAP" != "OK" ]; then
   exit 1
 fi
 
+
+#test GKE envrionment
+unset JAVA_OPTS TMPDIR GAE_MEMORY_MB HEAP_SIZE_MB JAVA_HEAP_OPTS JAVA_GC_OPTS JAVA_OPTS DBG_AGENT HEAP_SIZE_RATIO
+TMPDIR=/var/tmp
+KUBERNETES_MEMORY_LIMIT=20000000
```

Comment:
Also specify `HEAP_SIZE_RATIO`, so that the test is not flaky, when/if we change the default ratio.